andela/lotus-ah
# #160451590 Deploy base application to Heroku

#### What does this PR do?
- deploy the base application to Heroku

#### Description of Task to be completed?
 - create a continuous integration with continuous delivery pipeline
 - create a staging app `lotus-ah-staging`
 - create production app 'lotus-ah`
 - enable review app for every PR `lotus-staging-pr-id`
 - add deploy to .travis.yml

#### How should this be manually tested?
see [staging -  #lotus-ah-staging](https://lotus-ah-staging.herokuapp.com)

see [PR review app - #lotus-ah-staging](https://lotus-ah-staging-pr-32.herokuapp.com)

#### What are the relevant pivotal tracker stories?
[#160451590](https://www.pivotaltracker.com/story/show/160451590 "deploy app")
 
### Question
 NA

### Screenshot
<img width="1128" alt="screen shot 2018-10-18 at 3 18 14 am" src="https://user-images.githubusercontent.com/24890605/47127299-82fdfe80-d284-11e8-9bad-fbf888ee8d3a.png">
